public/mall/index.php

```diff
@@ -13,332 +19,133 @@
-          <a class="store-card-name" href="<?=url_for("/store/store-template");?>">Canon</a>
-        </div>
-
-        <div class="store-card">
-          <a href="<?=url_for("/store/store-template");?>"><img class="store-card-thumbnail" alt="image representation of a shop" src="../media/image/store_logos/dermalogica_logo.png"></a>
-          <a class="store-card-name" href="<?=url_for("/store/store-template");?>">Dermalogica</a>
-        </div>
+    // Dynamic loading of database
+    $products = read_csv("../../private/database/products.csv", true);
+    $stores = read_csv("../../private/database/stores.csv", true);
+    
+    // Sort products and stores from latest to oldest created time
+    usort($products, "compare_by_time");
+    usort($stores, "compare_by_time");
+    
+    include(SHARED_PATH . "/top.php");
 
-        <div class="store-card">
-          <a href="<?=url_for("/store/store-template");?>"><img class="store-card-thumbnail" alt="image representation of a shop" src="../media/image/store_logos/floralpunk_logo.png"></a>
-          <a class="store-card-name" href="<?=url_for("/store/store-template");?>">Floralpunk</a>
-        </div>
+?>
 
-        <div class="store-card">
-          <a href="<?=url_for("/store/store-template");?>"><img class="store-card-thumbnail" alt="image representation of a shop" src="../media/image/store_logos/hp_logo.png"></a>
-          <a class="store-card-name" href="<?=url_for("/store/store-template");?>">HP</a>
-        </div>
-      </div>
-      <button class="card-gallery-right-bttn"><i class="fas fa-angle-right"></i></button>
-    </section>
-  </main>
+    <div class="header-hero">
+        <img alt="Surprised person pointing at the Yabe logo" src="../media/image/hero.jpg">
+    </div>
+
+    <main id="mall-main">
+        <section class="card-gallery mall-home-section mall-home-section-gallery">
+            <h1 class="card-gallery-title">NEW PRODUCTS<span class="mall-home-section-heading-link"><a href="<?=url_for("/mall/browse/by-product/by-date.php");?>">MORE</a></span></h1>
+
+            <button class="card-gallery-left-bttn"><i class="fas fa-angle-left"></i></button>
+            <div class="card-gallery-content flex-container flex-justify-content-space-between flex-align-items-center overflow-hidden">
+                <?php
+                    
+                    // Loop counting the number of products until reaching 10 items
+                    $display_count = 0;
+                    while ($display_count < NEW_PRODUCTS_DISPLAY_NUM) {
+                        echo "<div class='product-card'>
+                        <a href='" . url_for("/store/store-template/product-detail") . "'><img alt='image of a product' src='../media/image/placeholder_262x250.png'></a>
+                        <div class='product-card-details'>
+                          <a class='product-card-title' href='" . url_for("/store/store-template/product-detail") . "'>" . $products[$display_count]["name"] . "</a>
```

Comment:
`url_for("/store/store-template/product-detail")` needs to have a `GET` query containing the product's ID so that @doanyennhi's dynamic product detail can work properly. E.g.
```php
url_for("/store/store-template/product-detail") . "?id={$products[$display_count]['id']}"
```